templates/admin/index.html.twig

```diff
@@ -0,0 +1,207 @@
+
+                                </div>
+                                <div class="card-body">
+                                    <h4 class="card-title">Demande n°</h4>
+                                    <p> Entreprise : </p>
+                                </div>
+
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col">
+                        <div class="cardClient">
+                            <div class="card-horizontal ">
+                                <div class="statut bg-success">
+
+                                </div>
+                                <div class="card-body">
+                                    <h4 class="card-title">Demande n°</h4>
+                                    <p> Entreprise : </p>
+                                </div>
+
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col">
+                        <div class="cardClient">
+                            <div class="card-horizontal">
+                                <div class="statut bg-success">
+
+                                </div>
+                                <div class="card-body">
+                                    <h4 class="card-title">Demande n°</h4>
+                                    <p> Entreprise : </p>
+                                    </div>
+
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="col">
+                        <div class="cardClient">
+                            <div class="card-horizontal">
+                                <div class="statut">
+
+                                </div>
+                                <div class="card-body">
+                                    <h4 class="card-title">Demande n°</h4>
+                                    <p> Entreprise : </p>
+                                </div>
+
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="col">
+                        <div class="cardClient">
+                            <div class="card-horizontal">
+                                <div class="statut">
+
+                                </div>
+                                <div class="card-body">
+                                    <h4 class="card-title">Demande n°</h4>
+                                    <p> Entreprise :</p>
+                                </div>
+
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+
```

Comment:
So could you tell me which bootstrap random template did you copypasted ?
I am not against bootstrap, may be usefull but please make a nice skeleton